tutorials/preprocessing/plot_80_boxy_processing.py

```diff
@@ -20,13 +20,23 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from itertools import compress
+import copy
 
 import mne
 
 
 boxy_data_folder = mne.datasets.boxy_example.data_path()
 boxy_raw_dir = os.path.join(boxy_data_folder, 'Participant-1')
-raw_intensity = mne.io.read_raw_boxy(boxy_raw_dir, verbose=True).load_data()
+raw_intensity = mne.io.read_raw_boxy(boxy_raw_dir, 'Ph', verbose=True).load_data()
+
+###separate data based on montages###
+mtg_a_indices = [i_index for i_index,i_label in enumerate(raw_intensity.info['ch_names']) if 'a01' in i_label]
```

Comment:
need another way to separate if not in channel name as asked for above

Reply:
yeah, adding an extra field to info would be easiest but I we aren't supposed to do that...

maybe add this information to raw_extras?

Reply:
lets just not add it for now, and add it later if needed

Reply:
okay sounds good. I'll resolve this once I push the changes related to label names.